server/src/main/java/org/elasticsearch/transport/TransportException.java

```diff
@@ -14,7 +14,7 @@
 import java.io.IOException;
 
 public class TransportException extends ElasticsearchException {
-    public TransportException(Throwable cause) {
+    protected TransportException(Throwable cause) {
```

Comment:
I would prefer to leave this bit out as well as the related changes.

Reply:
reverted